pkg/templates/openapi/ext.go

```diff
@@ -0,0 +1,320 @@
+package openapi
+
+const OpenAPIVersion = "3.0.3"
+
+const (
+	// Extension for UI.
+	ExtUI          = "x-walrus-ui"
+	ExtUIGroup     = "group"
+	ExtUIShowIf    = "show-if"
+	ExtUIHidden    = "hidden"
+	ExtUIImmutable = "immutable"
+	ExtUIWidget    = "widget"
+	ExtUISequence  = "sequence"
+
+	// Extension for original value.
+	ExtOriginal                = "x-walrus-original"
+	ExtOriginalLanguage        = "language"
+	ExtOriginalType            = "type"
+	ExtOriginalValueExpression = "value-expression"
```

Comment:
Do you think we can put everything in the same ext field like:
```yaml
x-walrus:
  ui:
    group: xxx
  original:
    type: xx
    language: xxx
```

How are language/type/value-expression being used?

Reply:
Prefer to keep the original and UI separated, so UI and backend could focus on the extension they care. Several extensions with the same prefix are common way. https://swagger.io/docs/specification/openapi-extensions/